### ISSUE TITLE
Checkra1n icon not showing on Beta 0.9.3

### ISSUE THREAD

**Issue by ghost** [checkra1n/BugTracker#312]
**What device + iOS version are you on?**
iPod 6th Generation, iOS 12.4.3

**What checkra1n version are you using?**
0.9.3

**What are the steps to reproduce the issue?**
Jailbreak with Checkra1n 0.9.3

**What do you expect, and what is happening instead?**
The icon to show up on the homescreen, instead it is not.

**Any other info, error logs, screenshots, ...?**
My tweaks still load so I know it worked. (I already have Cydia installed)

The icon shows up on 0.9.1 (The version I used before hand) and have never had a problem on 0.9.1, when I upgraded to 0.9.3 I had the problem. When using 0.9.1 after trying 0.9.3 it works, but 0.9.3 doesn't. So effectively 0.9.1 the icon shows, 0.9.3 which fixed others issues with it, caused the issue on mine.

**EDIT**

I tried 0.9.3.2, I can't even jailbreak, getting an error code -20.

On 0.9.5 it has changed to being delayed, the icon shows up several moments after the device boots into a jailbreak.

**Comment by gurkentier** [comment]
Try jailbreaking again entering DFU mode using the checkra1n GUI instructions. Fixed a lot of issues for me.

**Comment by GiladanoN** [comment]
very similar for me, took a while for the icon to show up under 0.9.3, currently unable to run cydia (probably a separate issue), if it'll persist i will make another bug report with details.

update:
this issue is told to be fixed under 0.9.3.2,
also also, got an answer to my probably unrelated cydia issue - https://github.com/checkra1n/BugTracker/issues/305#issuecomment-553615506

**Comment by Shadowedstalker** [comment]
#322 Will fix missing Icons on Springboard

**Comment by ghost** [comment]
> 
> 
> #322 Will fix missing Icons on Springboard

Yes I know UICache but as shown the Checkra1n icon not showing wasn't a UICache issue most of the time.

**Comment by ghost** [comment]
> 
> 
> very similar for me, took a while for the icon to show up under 0.9.3, currently unable to run cydia (probably a separate issue), if it'll persist i will make another bug report with details.
> 
> update:
> this issue is told to be fixed under 0.9.3.2,
> also also, got an answer to my probably unrelated cydia issue - [#305 (comment)](https://github.com/checkra1n/BugTracker/issues/305#issuecomment-553615506)

0.9.3.2 didn't seem to fix it for me, as I said 0.9.3 seemed to cause it for me as I didn't have the issue before.
